tests/components/music_assistant/common.py

```diff
@@ -0,0 +1,86 @@
+"""Provide common test tools."""
+
+from __future__ import annotations
+
+import json
+from typing import Any
+from unittest.mock import MagicMock
+
+from music_assistant_models.enums import EventType
+from music_assistant_models.player import Player
+from music_assistant_models.player_queue import PlayerQueue
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+def load_and_parse_fixture(fixture: str) -> dict[str, Any]:
+    """Load and parse a fixture."""
+    return json.loads(load_fixture(f"music_assistant/{fixture}.json"))
+
+
+async def setup_integration_from_fixtures(
+    hass: HomeAssistant,
+    music_assistant_client: MagicMock,
+) -> None:
+    """Set up MusicAssistant integration with fixture data."""
+    players = create_players_from_fixture()
+    music_assistant_client.players._players = {x.player_id: x for x in players}
+    player_queues = create_player_queues_from_fixture()
+    music_assistant_client.player_queues._queues = {
+        x.queue_id: x for x in player_queues
+    }
+    config_entry = MockConfigEntry(
+        domain="music_assistant", data={"url": "http://mock-music_assistant-server-url"}
+    )
```

Comment:
please use a constant for the url and use the DOMAIN for the domain. Also, we should add the unique id, as we have one